junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvArgumentsProvider.java

```diff
@@ -47,26 +50,40 @@ public void accept(CsvSource annotation) {
 		settings.getFormat().setQuoteEscape('\'');
 		settings.setEmptyValue(this.annotation.emptyValue());
 		settings.setAutoConfigurationEnabled(false);
+
 		CsvParser csvParser = new CsvParser(settings);
 		AtomicLong index = new AtomicLong(0);
 
+		DefaultConversionProcessor processor = getConversionProcessor();
+
 		// @formatter:off
-		return Arrays.stream(this.annotation.value())
-				.map(line -> {
-					String[] parsedLine = null;
-					try {
-						parsedLine = csvParser.parseLine(line + LINE_SEPARATOR);
-					}
-					catch (Throwable throwable) {
-						handleCsvException(throwable, this.annotation);
-					}
-					Preconditions.notNull(parsedLine,
-						() -> "Line at index " + index.get() + " contains invalid CSV: \"" + line + "\"");
-					return parsedLine;
-				})
-				.peek(values -> index.incrementAndGet())
-				.map(Arguments::of);
-		// @formatter:on
+        return Arrays.stream(this.annotation.value())
```

Comment:
Please use tabs instead of spaces for indentation.

Reply:
Sorry for mixing the indentations. I switched them back to tabs.